### PR TITLE
remove lower level debug logs from process sal

### LINKED ISSUE #575: remove lower level debug logs after adding more test coverage to core modules
Like e.g. process/nettools.

Once they're covered by tests, they can be removed.

### PR DESCRIPTION
**Pull request by sameh-farouk**
### Description

remove lower level debug logs from process sal

### Related Issues

closes #575 

### Checklist

- [x] [Pre-commit hook is installed](https://github.com/threefoldtech/js-ng#pre-commit) to do formatting checks before committing code...etc
- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [x] Code format and docstrings

### CODE REVIEW

File: jumpscale/sals/process/__init__.py [hunk 1]
```diff
@@ -187,37 +187,33 @@ def kill(proc, sig=signal.SIGTERM, timeout=5, sure_kill=False):
...
             j.logger.warning(f"Could not kill the process with pid: {proc.pid} with {sig}. Timeout: {timeout}")
             raise j.exceptions.Runtime(f"Could not kill process with pid {proc.pid}") from e
     except psutil.AccessDenied as e:
         # permission to perform an action is denied
-        j.logger.warning("the permission is denied")
         raise j.exceptions.Permission("Permission to perform this action is denied!") from e
     except psutil.NoSuchProcess:
         # Process no longer exists or Zombie (already dead)
-        j.logger.debug("Process is no longer exists or a Zombie (already dead)")
+        j.logger.warning(f"Process is no longer exists or a Zombie (already dead)")
```
Comment (abom) [inline jumpscale/sals/process/__init__.py:216]: can we remove this or keep it a debug log?

File: jumpscale/sals/process/__init__.py [hunk 2]
```diff
@@ -187,37 +187,33 @@ def kill(proc, sig=signal.SIGTERM, timeout=5, sure_kill=False):
...
         if sure_kill and sig != signal.SIGKILL and os.name != "nt":
             # SIGKILL not supported in windows
             # If a process gets this signal it must quit immediately and will not perform any clean-up operations
             proc.kill()
-            j.logger.debug("SIGKILL signal sent.")
+            # SIGKILL signal sent
             try:
                 proc.wait(1)
-                j.logger.debug(f"the process with PID: {proc.pid} was terminated with sig: {signal.SIGKILL}.")
+                j.logger.info(f"the process with PID: {proc.pid} was terminated with sig: {signal.SIGKILL}.")
```
Comment (abom) [inline jumpscale/sals/process/__init__.py:200]: can we remove this or keep it a debug log?

File: jumpscale/sals/process/__init__.py [hunk 3]
```diff
@@ -187,37 +187,33 @@ def kill(proc, sig=signal.SIGTERM, timeout=5, sure_kill=False):
         # If PID no longer exists return None immediately
         # If timeout exceeded and the process is still alive raise TimeoutExpired exception
         proc.wait(timeout=timeout)
-        j.logger.debug(f"the process with PID: {proc.pid} was terminated with sig: {sig}.")
+        j.logger.info(f"the process with PID: {proc.pid} was terminated with sig: {sig}.")
```
Comment (abom) [inline jumpscale/sals/process/__init__.py:190]: also this one

File: jumpscale/sals/process/__init__.py (hunk 1 above)
Comment (sameh-farouk) [reply jumpscale/sals/process/__init__.py]: removed

File: jumpscale/sals/process/__init__.py (hunk 2 above)
Comment (sameh-farouk) [reply jumpscale/sals/process/__init__.py]: removed

File: jumpscale/sals/process/__init__.py (hunk 3 above)
Comment (sameh-farouk) [reply jumpscale/sals/process/__init__.py]: removed

**Review by xmonader — APPROVED** [review]

**Pull request merged**